scripts/makeBob.py

```diff
@@ -0,0 +1,390 @@
+#!/usr/bin/env python3
+
+
+import phoebusgen
```

Comment:
Could you please move this  above line 13 with the related imports and remove this block of empty lines?